### PR TITLE
english corrections

### PR DESCRIPTION
**Pull request by Geeta-spec**
Signed-off-by: Geeta <geeta.garg@hpe.com>

### CODE REVIEW

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-The HPE 3PAR and Primera Container Storage Provider integrates as part of the [HPE CSI Driver for Kubernetes](../../csi_driver/index.md). The CSP abstract the data management capabilities of the array for use by Kubernetes. 
+The HPE 3PAR and Primera Container Storage Provider integrates as part of the [HPE CSI Driver for Kubernetes](../../csi_driver/index.md). The CSP abstracts the data management capabilities of the array for use by Kubernetes. 
 
 [TOC]
 
 ## Platform requirements
 
-Always check the corresponding CSI driver version in [compatibility and support](../../csi_driver/index.md#compatibility_and_support) and [SPOCK](#spock) for latest support matrix for the HPE 3PAR and Primera Container Storage Provider.
+Always check the corresponding CSI driver version in [compatibility and support](../../csi_driver/index.md#compatibility_and_support). Also check the [SPOCK](#spock) for latest support matrix for the HPE 3PAR and Primera Container Storage Provider.
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:9]: Should it be "Also check SPOCK for the latest.."

Comment (Geeta-spec) [reply docs/container_storage_provider/hpe_3par_primera/index.md]: Agree. Pushed the change.

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ The CSP requires access to a user with either `edit` or the `super` role. It's r
 
 ## StorageClass example
 
-A `StorageClass` is used to provision an HPE 3PAR or Primera Storage-backed persistent volume. Please see [using the HPE CSI Driver](../../csi_driver/using.md#base_storageclass_parameters) for additional base `StorageClass` examples like CSI snapshots and clones. 
+A `StorageClass` is used to provision an HPE 3PAR or Primera Storage-backed persistent volume. See [using the HPE CSI Driver](../../csi_driver/using.md#base_storageclass_parameters) for additional base `StorageClass` examples like CSI snapshots and clones.
```
Comment (drajen) [inline docs/container_storage_provider/hpe_3par_primera/index.md:35]: I don't think "Primera Storage" is a thing. Please remove.

**Comment by Geeta-spec** [comment]
> Is this still needed

Yes, please merge the change.

**Pull request closed without merging**